docker/Dockerfile.dev

```
@@ -0,0 +1,13 @@
+FROM esmvalgroup/esmvalcore:latest
```

```suggestion
FROM continuumio/miniconda3
```
same here, esmvalcore will be installed as any other dependency by conda